### PR TITLE
Update document editor styles (#180)

### LINKED ISSUE #180: ui: update document editor styles
Inputs now have labels and the new style

image

Buttons at the bottom were also updated, see #178

Figma: https://www.figma.com/file/P3X5qbxV5DZ0cM2arsmkrP/docs?node-id=122%3A1675

### PR DESCRIPTION
**Pull request by Y-k-Y**
Resolves #180

### CODE REVIEW

**Comment by talyguryn** [comment]
"new page" good for new pages but not for existed while editing. 

image

also "put above" does not updates but it is not for this pr

**Comment by Y-k-Y** [comment]
> "new page" good for new pages but not for existed while editing.
> 
> image
> 
> also "put above" does not updates but it is not for this pr

You are right. Can you suggest me the good label name pls?

**Comment by talyguryn** [comment]
> > "new page" good for new pages but not for existed while editing.
> > image
> > also "put above" does not updates but it is not for this pr
> 
> You are right. Can you suggest me the good label name pls?

why not "parent page"?

File: src/frontend/styles/components/writing.pcss [hunk 1]
```diff
@@ -17,16 +16,40 @@
...
+      width: 100%;
+      border: none;
+      border-radius: 8px;
+      background-color: #F3F6F8;
+      box-sizing: border-box;
+      appearance: none;
+    }
+
+    select {
+      padding-right: 36px;
```
Comment (neSpecc) [inline src/frontend/styles/components/writing.pcss:48]: padding right seems too big.

Stage:
image

Figma: 
image

File: src/frontend/styles/components/writing.pcss [hunk 2]
```diff
@@ -17,16 +16,40 @@
...
+      background-image: url("data:image/svg+xml,%3Csvg width='8' height='5' viewBox='0 0 8 5' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M7.16696 0.613295C7.04205 0.489128 6.87308 0.419434 6.69696 0.419434C6.52083 0.419434 6.35187 0.489128 6.22696 0.613295L3.83362 2.9733L1.47362 0.613295C1.34872 0.489128 1.17975 0.419434 1.00362 0.419434C0.827501 0.419434 0.658533 0.489128 0.533625 0.613295C0.471139 0.675271 0.421543 0.749005 0.387697 0.830244C0.353851 0.911484 0.336426 0.998621 0.336426 1.08663C0.336426 1.17464 0.353851 1.26177 0.387697 1.34301C0.421543 1.42425 0.471139 1.49799 0.533625 1.55996L3.36029 4.38663C3.42227 4.44911 3.496 4.49871 3.57724 4.53256C3.65848 4.5664 3.74562 4.58383 3.83362 4.58383C3.92163 4.58383 4.00877 4.5664 4.09001 4.53256C4.17125 4.49871 4.24498 4.44911 4.30696 4.38663L7.16696 1.55996C7.22944 1.49799 7.27904 1.42425 7.31288 1.34301C7.34673 1.26177 7.36416 1.17464 7.36416 1.08663C7.36416 0.998621 7.34673 0.911484 7.31288 0.830244C7.27904 0.749005 7.22944 0.675271 7.16696 0.613295Z' fill='black'/%3E%3C/svg%3E%0A");
+      background-repeat: no-repeat;
+      background-position: right 10px center;
+    }
   }
 }
```
Comment (neSpecc) [inline src/frontend/styles/components/writing.pcss:54]: Also, lets update the `focus` stage:

Stage: 
image

Figma: 
image

**Review by neSpecc — CHANGES_REQUESTED** [review]

File: src/frontend/styles/components/writing.pcss (hunk 2 above)
Comment (Y-k-Y) [reply src/frontend/styles/components/writing.pcss:44]: Fixed

File: src/frontend/styles/components/writing.pcss (hunk 1 above)
Comment (Y-k-Y) [reply src/frontend/styles/components/writing.pcss]: The caret icon was bigger than Figma so downsized it

File: src/frontend/styles/components/writing.pcss [hunk 3]
```diff
@@ -17,16 +16,40 @@
...
+    label {
+      display: block;
+      margin-bottom: 6px;
     }
-  }
 
-  select {
-    max-width: 100px;
-    vertical-align: bottom;
+    select, input {
```
[the remark this answers is not in the document]
Comment (Y-k-Y) [reply src/frontend/styles/components/writing.pcss]: Fixed

**Comment by neSpecc** [comment]
Pls, update the brunch

File: src/frontend/styles/vars.pcss [hunk 4]
```diff
@@ -73,6 +75,47 @@
...
+    appearance: none;
+    background-image: url("data:image/svg+xml,%3Csvg width='8' height='5' viewBox='0 0 8 5' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M7.16696 0.613295C7.04205 0.489128 6.87308 0.419434 6.69696 0.419434C6.52083 0.419434 6.35187 0.489128 6.22696 0.613295L3.83362 2.9733L1.47362 0.613295C1.34872 0.489128 1.17975 0.419434 1.00362 0.419434C0.827501 0.419434 0.658533 0.489128 0.533625 0.613295C0.471139 0.675271 0.421543 0.749005 0.387697 0.830244C0.353851 0.911484 0.336426 0.998621 0.336426 1.08663C0.336426 1.17464 0.353851 1.26177 0.387697 1.34301C0.421543 1.42425 0.471139 1.49799 0.533625 1.55996L3.36029 4.38663C3.42227 4.44911 3.496 4.49871 3.57724 4.53256C3.65848 4.5664 3.74562 4.58383 3.83362 4.58383C3.92163 4.58383 4.00877 4.5664 4.09001 4.53256C4.17125 4.49871 4.24498 4.44911 4.30696 4.38663L7.16696 1.55996C7.22944 1.49799 7.27904 1.42425 7.31288 1.34301C7.34673 1.26177 7.36416 1.17464 7.36416 1.08663C7.36416 0.998621 7.34673 0.911484 7.31288 0.830244C7.27904 0.749005 7.22944 0.675271 7.16696 0.613295Z' fill='black'/%3E%3C/svg%3E%0A");
+    background-repeat: no-repeat;
+    background-position: right 15px center;
+
+    &:focus {
```
Comment (TatianaFomina) [inline src/frontend/styles/vars.pcss:91]: there's a layout shift on focus, can we fix it?

https://user-images.githubusercontent.com/31101125/175306180-7b6be8d5-21f6-4716-9f68-d72aea89b7cb.mov

File: src/backend/views/pages/form.twig [hunk 5]
```diff
@@ -13,14 +13,18 @@
       }
     </textarea>
     <header class="writing-header">
-      <span class="writing-header__left">
-        <span>
-        New Page at the
+      <div class="writing-header__left">
```
Comment (TatianaFomina) [inline src/backend/views/pages/form.twig:16]: is this class name still actual?

Comment (TatianaFomina) [reply src/backend/views/pages/form.twig:16]: why `left`?

File: src/frontend/styles/components/writing.pcss [hunk 6]
```diff
@@ -17,16 +16,30 @@
...
+    gap: 10px;
 
-    & span {
-      margin-right: 10px;
+    @media(--mobile) {
+      flex-flow: column;
+    }
+
+    & > * {
+      flex: 0 1 33.3%;
```
Comment (TatianaFomina) [inline src/frontend/styles/components/writing.pcss:28]: this could be replaced with `flex: 1 0;`

File: src/backend/views/pages/form.twig (hunk 5 above)
Comment (Y-k-Y) [reply src/backend/views/pages/form.twig]: Renamed to `inner-container` since `left` has no meaning now

File: src/frontend/styles/vars.pcss [hunk 7]
```diff
@@ -73,6 +75,47 @@
...
+    border-radius: 8px;
+    background-color: var(--color-input-primary);
+    box-sizing: border-box;
+    appearance: none;
+    background-image: url("data:image/svg+xml,%3Csvg width='8' height='5' viewBox='0 0 8 5' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M7.16696 0.613295C7.04205 0.489128 6.87308 0.419434 6.69696 0.419434C6.52083 0.419434 6.35187 0.489128 6.22696 0.613295L3.83362 2.9733L1.47362 0.613295C1.34872 0.489128 1.17975 0.419434 1.00362 0.419434C0.827501 0.419434 0.658533 0.489128 0.533625 0.613295C0.471139 0.675271 0.421543 0.749005 0.387697 0.830244C0.353851 0.911484 0.336426 0.998621 0.336426 1.08663C0.336426 1.17464 0.353851 1.26177 0.387697 1.34301C0.421543 1.42425 0.471139 1.49799 0.533625 1.55996L3.36029 4.38663C3.42227 4.44911 3.496 4.49871 3.57724 4.53256C3.65848 4.5664 3.74562 4.58383 3.83362 4.58383C3.92163 4.58383 4.00877 4.5664 4.09001 4.53256C4.17125 4.49871 4.24498 4.44911 4.30696 4.38663L7.16696 1.55996C7.22944 1.49799 7.27904 1.42425 7.31288 1.34301C7.34673 1.26177 7.36416 1.17464 7.36416 1.08663C7.36416 0.998621 7.34673 0.911484 7.31288 0.830244C7.27904 0.749005 7.22944 0.675271 7.16696 0.613295Z' fill='black'/%3E%3C/svg%3E%0A");
```
Comment (robonetphy) [inline src/frontend/styles/vars.pcss:87]: can we put this inside the SVG folder

**Review by robonetphy — CHANGES_REQUESTED** [review]
the word is cutting a little bit 
image.
I guess you need to add `line-height:18px`

**Review by robonetphy — CHANGES_REQUESTED** [review]
image
Pls do the `line-height` change with `input` too

**Review by robonetphy — APPROVED** [review]

**Review by neSpecc — APPROVED** [review]
1. Input heights

**Pull request merged**